connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerder.java

```diff
@@ -424,6 +435,9 @@ private void updateConnectorTasks(String connName) {
         if (!worker.isRunning(connName)) {
             log.info("Skipping update of connector {} since it is not running", connName);
             return;
+        } else if (configState.targetState(connName) != TargetState.STARTED) {
+            log.info("Skipping update of connector {} since its target state is {}", connName, configState.targetState(connName));
```

Comment:
```suggestion
            log.info("Skipping update of connector tasks for connector {} since its target state is {}", connName, configState.targetState(connName));
```
nit: `Skipping update of connector` sounds ambiguous

Reply:
I agree with this, but "connector tasks for connector" is also a bit verbose--I've updated this statement and the one above it to "Skipping update of tasks for connector".